vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -193,52 +200,54 @@ type internal FSharpCheckerWorkspaceServiceFactory
-                             CodeSenseDelay = 100,
-                             ShowDropDownOptions = true)>]
-    internal FSharpPackage() =
+
+[<Guid(FSharpCommonConstants.packageGuidString)>]
+[<ProvideLanguageService 
+    (   languageService = typeof<FSharpLanguageService>
+    ,   strLanguageName = FSharpCommonConstants.FSharpLanguageName
+    ,   languageResourceID = 100
+    ,   MatchBraces = true
+    ,   MatchBracesAtCaret = true
+    ,   ShowCompletion = true
+    ,   ShowMatchingBrace = true
+    ,   ShowSmartIndent = true
+    ,   EnableAsyncCompletion = true
+    ,   QuickInfo = true
+    ,   DefaultToInsertSpaces = true
+    ,   CodeSense = true
+    ,   DefaultToNonHotURLs = true
+    ,   EnableCommenting = true
+    ,   CodeSenseDelay = 100
+    ,   ShowDropDownOptions = true )>]
+type internal FSharpPackage() =
     inherit AbstractPackage<FSharpPackage, FSharpLanguageService>()
     
     override this.RoslynLanguageName = FSharpCommonConstants.FSharpLanguageName
 
     override this.CreateWorkspace() = this.ComponentModel.GetService<VisualStudioWorkspaceImpl>()
 
-    override this.CreateLanguageService() = 
-        FSharpLanguageService(this)        
+    override this.CreateLanguageService() = FSharpLanguageService (this)        
 
     override this.CreateEditorFactories() = Seq.empty<IVsEditorFactory>
 
     override this.RegisterMiscellaneousFilesWorkspaceInformation(_) = ()
     
-and 
-    [<Guid(FSharpCommonConstants.languageServiceGuidString)>]
-    [<ProvideLanguageExtension(typeof<FSharpLanguageService>, ".fs")>]
-    [<ProvideLanguageExtension(typeof<FSharpLanguageService>, ".fsi")>]
-    [<ProvideLanguageExtension(typeof<FSharpLanguageService>, ".fsx")>]
-    [<ProvideLanguageExtension(typeof<FSharpLanguageService>, ".fsscript")>]
-    [<ProvideLanguageExtension(typeof<FSharpLanguageService>, ".ml")>]
-    [<ProvideLanguageExtension(typeof<FSharpLanguageService>, ".mli")>]
-    [<ProvideEditorExtension(FSharpCommonConstants.editorFactoryGuidString, ".fs", 97)>]
-    [<ProvideEditorExtension(FSharpCommonConstants.editorFactoryGuidString, ".fsi", 97)>]
-    [<ProvideEditorExtension(FSharpCommonConstants.editorFactoryGuidString, ".fsx", 97)>]
-    [<ProvideEditorExtension(FSharpCommonConstants.editorFactoryGuidString, ".fsscript", 97)>]
-    [<ProvideEditorExtension(FSharpCommonConstants.editorFactoryGuidString, ".ml", 97)>]
-    [<ProvideEditorExtension(FSharpCommonConstants.editorFactoryGuidString, ".mli", 97)>]
+
+
+and [<Guid(FSharpCommonConstants.languageServiceGuidString)>]
+    [<ProvideService (typeof<FSharpLanguageService>,ServiceName="F# Language Service")>]
+    [<ProvideLanguageExtension (typeof<FSharpLanguageService>, ".fs")>]
+    [<ProvideLanguageExtension (typeof<FSharpLanguageService>, ".fsi")>]
+    [<ProvideLanguageExtension (typeof<FSharpLanguageService>, ".fsx")>]
+    [<ProvideLanguageExtension (typeof<FSharpLanguageService>, ".fsscript")>]
```

Comment:
Please don't reformat code as part of PRs - minimize the diffs please and do reformatting in a separate PR.  Large diffs makes it hard to review.